Yosemite/Yosemite/Stores/RefundStore.swift

```diff
@@ -36,8 +36,8 @@ public class RefundStore: Store {
             createRefund(siteID: siteID, orderID: orderID, refund: refund, onCompletion: onCompletion)
         case .retrieveRefund(let siteID, let orderID, let refundID, let onCompletion):
             retrieveRefund(siteID: siteID, orderID: orderID, refundID: refundID, onCompletion: onCompletion)
-        case .retrieveRefunds(let siteID, let orderID, let refundIDs, let onCompletion):
-            retrieveRefunds(siteID: siteID, orderID: orderID, refundIDs: refundIDs, onCompletion: onCompletion)
+        case .retrieveRefunds(let siteID, let orderID, let refundIDs, let deleteStaleRefunds, let onCompletion):
+            retrieveRefunds(siteID: siteID, orderID: orderID, refundIDs: refundIDs, deleteStaleRefunds: deleteStaleRefunds, onCompletion: onCompletion)
         case .synchronizeRefunds(let siteID, let orderID, let pageNumber, let pageSize, let onCompletion):
```

Comment:
deleting stale refunds made me wonder how we handle deleted refunds in `synchronizeRefunds`, and it looks like it's not even used 🤔

Reply:
Yeah, the only action used `retrieveRefunds` do you think we should delete the rest of them?